### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by mum33n**
Added navbar and working on wallet provider

### CODE REVIEW

**Review by ivan-ngchakming — COMMENTED** [review]
It seems you accidentally modified some lines in the Dummy page other than just adding the Navbar component. 

Also, it seems you didn't implement the wallet connect function?

File: frontend/src/components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames'
+import React from 'react'
+import { BsWallet } from "react-icons/bs"
+import cn from 'classnames'
+
+function Navbar() {
+  return (
+    <div className='px-5 py-3 bg-slate-900 items-center shadow-lg mx-auto flex justify-between'>
+        <a href='/'><h1 className='text-white font-bold text-2xl'>ZILLIQA</h1></a>
+        <button className={
```
Comment (ivan-ngchakming) [inline frontend/src/components/Navbar.tsx]: Use the Button component instead

Comment (mum33n) [reply frontend/src/components/Navbar.tsx]: I have corrected it

Comment (ivan-ngchakming) [reply frontend/src/components/Navbar.tsx]: You need to push your commits when you have made any new changes

**Review by mum33n — COMMENTED** [review]
added the connect and disconnect buttons

**Pull request merged**